backend/src/main/kotlin/dev/pango/apollo/backend/modules/userauth/data/repository/UserRepositoryImpl.kt

```diff
@@ -10,10 +10,14 @@ import org.ktorm.dsl.*
 import org.ktorm.entity.*
 
 class UserRepositoryImpl : UserRepository {
+
     override suspend fun searchUser(id: Int): User? =
         database.sequenceOf(UserTable)
             .find { user -> user.id eq id }
 
+    override suspend fun findAllUsers(): Set<User> =
+        database.sequenceOf(UserTable).toSet()
+
     override suspend fun updateUser(
```

Comment:
Should return Either<Failure, User>, User updated, and change to updateUser(user: User)